coc/client.py

```diff
@@ -257,9 +256,15 @@ async def login(self, email: str, password: str) -> None:
         """
         self.http = http = self._create_client(email, password)
         await http.create_session(self.connector, self.timeout)
-        await http.initialise_keys()
-        self._create_holders()
 
+        try:
+            await http.initialise_keys()
+        except Exception as error:
```

Comment:
Why not let the error from http propagate and log it back in http that it failed?

Reply:
Having a hard time navigating with my phone but I think that this should not be there. I think I added the try except to narrow down where a bug was coming from.